x-pack/plugins/endpoint/common/types.ts

```diff
@@ -656,7 +656,7 @@ interface HostPolicyResponseConfigurationStatus {
  * Information about the applying of a policy to a given host
  */
 export interface HostPolicyResponse {
```

Comment:
Above, on line 647 - I think that the `actions` are not longer permissive - meaning: we don't allow for "unkonwn" actions to be defined. Is that true? If so, then we should remove lines 645-647 in the `HostPolicyResponseActions` above. 

If that is still the case that we should accept actions that Kibana does not have a type for, then I think the generators and/or tests need to account for that as well (_I think the generator only returns know actions_)

Reply:
Yes, I can remove it. But do think we should still make it partial. If not all are applicable in all cases.

Reply:
yeah, I was only referring to the list being permissive. I still agree that when this list is used in the Policy Response type, it should be `Partial<>`